### DISCUSSION TITLE
Adding GEOM-GCN

### DISCUSSION THREAD

**Discussion by RasmusOrsoe** [pyg-team/pytorch_geometric#3026] [Ideas]
Hi!

Came across GEOM-GCN when I was taking a peak at 

https://pytorch-geometric.readthedocs.io/en/latest/modules/utils.html#torch_geometric.utils.homophily 

I think the geometric embedding scheme proposed in the paper, where one can choose between embedding methods, could be very useful as a pytorch module for those of us doing geometric learning.. :-)

I can't find it implemented in pytorch geometric. Is there a reason why its not implemented?

Thanks

**Comment by rusty1s** [comment]
There is no specific reason it is not integrated in PyG, and I would love to include it. Let me know if you have any interest in working on this together.
